ui/app/pages/permissions-connect/choose-account/choose-account.component.js

```diff
@@ -39,10 +83,19 @@ export default class ChooseAccount extends Component {
             return (
               <div
                 key={`permissions-connect-choose-account-${index}`}
-                onClick={ () => selectAccount(address) }
+                onClick={ () => this.handleAccountClick(address) }
                 className="permissions-connect-choose-account__account"
               >
                 <div className="permissions-connect-choose-account__account-info-wrapper">
+                  { accounts.length > 1
+                    ? (
+                      <CheckBox
+                        className="permissions-connect-choose-account__list-check-box"
+                        checked={ selectedAccounts.has(address) }
+                      />
+                    )
+                    : null
```

Comment:
This results in the selection state being hidden when there is only one account. The "Next" button is mysteriously disabled unless you click the single available account, after which it's enabled, all without any visual indication as to why.

We should probably just leave the checkbox in even for a list of one.

Reply:
Done in 9c834080e